src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/TestFactory.cs

```diff
@@ -424,8 +424,7 @@ private void AddHackerUser(List<ProCoSysProject> commonProCoSysProjects)
                         Permissions.LIBRARY_FUNCTIONAL_ROLE_READ,
                         Permissions.USER_READ,
                         Permissions.IPO_READ,
-                        Permissions.IPO_SIGN,
```

Comment:
I dont' get this. I don't understand why integrationtests need both UserType.Signer, UserType.Completer and UserType.Accepter. They have exact same privileges, and can therefor do the same (access same endpoints).
Add a TECH, and we can discuss it after completing this PR

Reply:
Agree

Reply:
Product Backlog Item 81792: TECH - IPO - Clean up integration tests